### DISCUSSION TITLE
How to construct a conduit key?

### DISCUSSION THREAD

**Discussion by Vincent18zt** [ProjectOpenSea/seaport#544] [Q&A] [answered]
It seems that no construct methods of conduit key are illustrated in documents.
I try to construct a conduit key which the first 20 bytes are my address and the after 12 bytes are random characters, but finally failed to execute `deployConduit` on chain.
So how can I make a conduit key?

**Comment by pamungkaski** [comment, marked as answer]
https://etherscan.io/address/0x00000000F9490004C11Cef243f5400493c00Ad63#writeContract
You can use createConduit functions here in the conduitController to create a new conduit key

**Reply by stefstef800** [reply]
thank you for the answer, do you think you could help clarify. Because in the "createConduit-functions" it says that a conduit key is required to create a new conduit. How does creating a new key work?

<img width="763" alt="Screen Shot 2022-07-20 at 5 28 08 PM" src="https://user-images.githubusercontent.com/88905213/180085878-c268e85c-9b22-48ed-b78d-15ea19bad71a.png">
<img width="763" alt="Screen Shot 2022-07-20 at 5 30 26 PM" src="https://user-images.githubusercontent.com/88905213/180085881-37f0e3b8-977c-4762-ba1f-2cbd292049dd.png">

**Reply by arr00** [reply]
A conduit key is 32 bytes long where the first 20 bytes are the address of the creator, and the remaining 12 bytes can be anything. For example, if the creator is `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266`, a valid conduit key is `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266000000000000000000000000`. Please reply to this thread if you have any further issues.

**Reply by stefstef800** [reply]
thank you so much for the great explanation!
